testing_scripts/run_lfind.py

```diff
@@ -0,0 +1,49 @@
+import os
+import subprocess
+
+# This file includes functions in charge of:
```

Comment:
Maybe you already have this,  but could you either point me to in the scriptinfo.txt the usage of this file or maybe add it if its missing?

Reply:
What file are you referring to? The run_lfind.py? I had included a comment at the top that explains what happens in the file, do you think that should be more explanation? I had just wanted to have the code that is charge of running make on all of the lfind folders in a file. Maybe I should change that?